lib/Appium.pm

```diff
@@ -194,7 +194,7 @@ has 'touch_actions' => (
     is => 'ro',
     lazy => 1,
     init_arg => undef,
-    handles => [ qw/tap/ ],
+    handles => [ qw/tap swipe wait moveTo release press longPress/ ],
```

Comment:
prefer underscore_case for compound words instead of camelCase to match the convention in the rest of the repository